### DISCUSSION TITLE
sytnax error, unexpected 'use'

### DISCUSSION THREAD

**Discussion by ardabalkandev** [dompdf/dompdf#2999] [General]
Hello,

I installed all necessery files and I couldn't run the codes because of this error: "syntax error, unexpected 'use' "

The codes:

```
      require_once 'vendor/autoload.php';

      use Dompdf\Dompdf;

      $dompdf=new Dompdf();

      $html='bla bla';
```

**Comment by bsweeney** [comment]
Is this contained within another class definition? The `use` statement must be in the outermost scope of the executing script.
